lib/metadata/bucketfile/backend.js

```diff
@@ -0,0 +1,402 @@
+import { errors } from 'arsenal';
+import level from 'level';
+import multilevel from 'multilevel';
+import sublevel from 'level-sublevel';
+import net from 'net';
+import BucketInfo from '../BucketInfo';
+import arsenal from 'arsenal';
+import events from 'events';
+import assert from 'assert';
+import cluster from 'cluster';
+import fs from 'fs';
+import constants from '../../../constants';
```

Comment:
Can you please re-order the modules in this order?
1) core modules
2) dependency modules
3) local modules